src/main/java/org/mskcc/cbio/oncokb/web/rest/slack/DropdownEmailOption.java

```diff
@@ -159,6 +174,11 @@ public DropdownEmailOptionBuilder category(EmailCategory category) {
             return this;
         }
 
+        public DropdownEmailOptionBuilder specificLicense(LicenseType licenseType) {
```

Comment:
some emails should only show up for certain license type. i.e. domain clarification only for academic, license options only for commercial, research in commercial, hospital